src/Behat/Gherkin/Keywords/KeywordsInterface.php

```diff
@@ -100,4 +100,12 @@ public function getButKeywords();
      * @return string
      */
     public function getStepKeywords();
+
+
+    /**
+     * Returns all rule keywords (splitted by "|").
+     *
+     * @return string
+     */
+    public function getRuleKeywords();
```

Comment:
[BC] We should define a new interface that extends KeywordsInterface and adds this method